widgetssdk/src/main/java/com/glia/widgets/call/CallActivityIntentHelper.kt

```diff
@@ -0,0 +1,41 @@
+package com.glia.widgets.call
+
+import android.content.Context
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import com.glia.androidsdk.Engagement
+import com.glia.widgets.GliaWidgets
+import com.glia.widgets.core.configuration.GliaSdkConfiguration
+
+class CallActivityIntentHelper {
+
+    companion object {
+        @JvmStatic
+        fun createIntent(context: Context, configuration: Configuration?): Intent {
```

Comment:
`configuration` property should be non-null since we're checking for that inside this function. I think this `configuration ?: throw NullPointerException("Configuration can't be null")` check could be moved to the caller side if needed.